### DISCUSSION TITLE
Issue with Excessive Filament Retraction after Pause and Filament Change in Orca Slicer

### DISCUSSION THREAD

**Discussion by PoberezhnyiD** [OrcaSlicer/OrcaSlicer#7372] [General]
Issue with Excessive Filament Retraction after Pause and Filament Change in Orca Slicer

I'm experiencing a significant issue with excessive filament retraction after resuming a print from a pause and during filament changes in Orca Slicer. This problem occurs across all versions of Orca Slicer that I've tried (at least five versions) and affects multiple 3D printers.

Symptoms:
- After I pause the print, the filament stops correctly, with no excessive retraction.
- However, when I resume the print (press "play"), there is a very strong and fast retraction before the print continues, which leads to improper filament positioning in the extruder.
- This same problem occurs during filament changes: after the filament is replaced, there is an excessive retraction, making it difficult to resume printing properly.
- Due to this excessive retraction, the filament does not return to its correct position, causing extrusion issues and, ultimately, defects in the print.

What I've Tried So Far:
1. Adjusted and reduced retraction settings in the filament and extruder settings.
2. Used custom G-code for the pause to prevent retraction during pauses:
   ```gcode
   G1 E0 F1200 ; Keep the filament in place without retraction
   M0 ; Pause
   G1 E0 F1200 ; Keep the filament in place after resuming
   ```
3. Enabled the "Use firmware retraction" option, but this did not resolve the issue either.
4. Tested several different versions of Orca Slicer, yet the problem persists.

Additional Information:
- This issue occurs not only on my Artillery Sidewinder X2 printer but also on a **Creality K1 Max**, indicating that it's not specific to a single printer model.
- When using a different slicer, specifically FullSan in 400, everything works perfectly, with no excessive retraction during pauses or filament changes.
- Printer firmware: Marlin (legacy) on Artillery Sidewinder X2, which supports `G10` and `G11` commands for retraction control.

Request to the Community:
I would greatly appreciate any advice or recommendations. Is there a specific setting or option that I need to disable or adjust to prevent this excessive retraction after pausing and during filament changes? If anyone has encountered a similar issue and found a workaround, please share your experience.

**Comment by braeislost** [comment]
Did you ever find a solution to this? Not much filament is being laid down after I resume, it seems like it resumes with the nozzle too high or is retracting too much. Printer is an Ender 3 V3 SE.

**Reply by Colsifer** [reply]
I had this issue on my SV06 and commented below on how I fixed it. Hopefully it helps you as well!

**Comment by RonAsor** [comment]
I've been running through this problem as well and was looking for any lead about this issue... I'm currently trying to find a workaround for this on my ender 5 pro printer.
I noticed this when the filament  was going towards the prime tower that it was fed correctly up to the nozzle, but when it resumes there's a significant distance between the filament and the nozzle, even when you resume the print with the same filament it does this... kind of peculiar

**Reply by RonAsor** [reply]
@PoberezhnyiD A thing I just noticed on my side for the custom gcode:
I've been using G91 and G90 to set the head height a little bit higher so I could purge the filament manually before it starts the purging tower sequence.
after setting G90, I didn't set M83 (which was my issue), so when it lifted the head from the bed, the first extrusion command was E0.9112.. so it probably did a substraction from the previous command to that one which caused my extrusion pull.
setting M83 after G90 solved this for me (although still having trouble to fix the filament to the bed, but the extrusion path even with the junk seems finely fed)

I wish we could set the purge tower brim to use the first/any material, it might make the changing process a bit less troublesome so it would have more potential to stick to the surface..

**Comment by Colsifer** [comment]
I had the same issue on my SV06 to the point where it actually pushes the filament all the way out of the extruder and can't grip it anymore when it tries to print again. I had to manually shove it back in and just hope it extrudes enough distance on the wipe tower to actually start extruding filament again. If anyone is still looking for an answer to this, what fixed it for me was changing the Cooling tube position and Filament parking positon values in the printer settings (as shown in image).
![MultimaterialParameters](https://github.com/user-attachments/assets/fdd55219-8f64-4581-afc9-e36d380a9c41)
Orca's profile for the SV06 had it set to something around 90mm by default, when it should have been closer to 15mm. Setting them to the 15mm and 16mm values in the image seems to have fixed the issue completely. I recommend googling what these parameters should be for your printer, and/or guess and check with a simple test print.
